style/middleware/_middleware.filepicker.scss

```diff
@@ -88,6 +88,10 @@ $grid-breakpoints: (
   @extend .modal-title;
 }
 
+.moodle-dialogue-hd h1 {
+  font-size: $global-font-size-h2;
```

Comment:
Is there a reason we're making a h1 the size of a h2?

Reply:
This is to fit the "x" close button within the popup. Currently, the Confirmation dialog/popup has inline style/width added to it from the code with an ID selector which I can't override, hence tweaking this the other way round by making the heading slightly smaller to make the pop-up look neat.